src/container.ts

```diff
@@ -609,6 +617,17 @@ export class Container {
       throw new Error('Attempted to construct a container from a Container object.');
     }
 
+    const defaultResourceSpec: ContainerResources = {
+      cpu: {
+        request: Cpu.millis(1000),
+        limit: Cpu.millis(1500),
+      },
+      memory: {
+        request: Size.mebibytes(1769),
```

Comment:
I've thought about this more and I think this might be a bit much for a default. It will make it hard to deploy applications to a local cluster.

I think that the separation between `request` and `limit` gives us the ability to provide a good balance between developer friendly, and safe.

```suggestion
        request: Size.mebibytes(512),
```

Reply:
I see, will make the update. Do you want to reduce the CPU too for request?

Reply:
Nah. CPU is ok.